tailwind.config.js

```diff
@@ -119,6 +119,6 @@ module.exports = {
     require("daisyui"),
     require("@tailwindcss/typography"),
     require("@tailwindcss/container-queries"),
-    require("tailwind-scrollbar"),
+    require("tailwindcss-scrollbar"),
```

Comment:
Is there a reason you changed this from tailwind-scrollbar to tailwindcss-scrollbar? The package that is used here is this one:
https://www.npmjs.com/package/tailwind-scrollbar

Reply:
This is because an error occurs during build.